src/main/java/com/test/amazons/game/Board.java

```diff
@@ -0,0 +1,60 @@
+package com.test.amazons.game;
+
+import com.test.amazons.model.Colour;
+import com.test.amazons.model.Position;
+import com.test.amazons.model.Tile;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Component;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class Board {
+
+    // Elwyn says to start off with a 6x6 board
+
+    private final Integer rows;
+    private final Integer columns;
```

Comment:
int

Reply:
explain pls

Reply:
*grumbles* Kotlin doesn't have primitive types, we wouldn't be having this conversation *grumbles some more*